### PR TITLE
Update timezone language

### PR DESCRIPTION
**Pull request by efischer19**
# [TNL-6051](https://openedx.atlassian.net/browse/TNL-6051)

@catong and @yro to review, when you have a chance.

### CODE REVIEW

**Comment by efischer19** [comment]
Screenshot of previous wording, from my devstack:
screen shot 2016-12-14 at 12 50 45 pm

File: problem/9cee77a606ea4c1aa5440e0ea5d0f618.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <p>Most courses have interactive questions that test your knowledge like the one below.  They can be part of a learning sequence or an exam.  Notice the visual feedback.  Go ahead, try it out!</p>
   <p>Questions which are part of assignments or exams may have due dates - the last possible time you can submit an assignment for grading. Once this time has passed, you will not be able to get credit for any incomplete problems in the assignment.</p>
   <p>If an assignment has a due date, you can see the due date in the accordion on the side. (This demo course does not have any due date.) If no due date is displayed, the assignment can be turned in at any time.</p>
-  <p>All assignment due dates are listed in UTC (the international time standard). Please <a href="http://www.timeanddate.com/worldclock/">look up your time zone</a> in order to understand what time your assignments are due according to your clock. EdX cannot offer any extensions.</p>
+  <p>All assignment due dates are listed in the user's specified timezone, defaulting to UTC (the international time standard) if one has not been chosen. EdX cannot offer any extensions.</p>
```
Comment (catong) [inline problem/9cee77a606ea4c1aa5440e0ea5d0f618.xml]: @efischer19 @yro Is it still true that the assignment dates will default to UTC if a user has not specified a time zone? I thought browser time zone would be used for assignment dates if a time zone wasn't set in account settings.

Comment (catong) [inline problem/9cee77a606ea4c1aa5440e0ea5d0f618.xml]: @efischer19 If you don't mind, can we also make a couple of changes to the paragraph previous to the one where time zone is mentioned.
"...you can see the due date in the accordion on the side" -> "you can see the due date in the sidebar."
"This demo course does not have any due date." -> "This demo course does not have any assignments with due dates."

Comment (efischer19) [inline problem/9cee77a606ea4c1aa5440e0ea5d0f618.xml]: Definitely, that's why I tagged our neighborhood doc expert 😃 

New commit incoming...

File: problem/9cee77a606ea4c1aa5440e0ea5d0f618.xml [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
   <p>Questions which are part of assignments or exams may have due dates - the last possible time you can submit an assignment for grading. Once this time has passed, you will not be able to get credit for any incomplete problems in the assignment.</p>
-  <p>If an assignment has a due date, you can see the due date in the accordion on the side. (This demo course does not have any due date.) If no due date is displayed, the assignment can be turned in at any time.</p>
-  <p>All assignment due dates are listed in UTC (the international time standard). Please <a href="http://www.timeanddate.com/worldclock/">look up your time zone</a> in order to understand what time your assignments are due according to your clock. EdX cannot offer any extensions.</p>
+  <p>If an assignment has a due date, you can see the due date in the sidebar. (This demo course does not have any assignments with due dates.) If no due date is displayed, the assignment can be turned in at any time.</p>
+  <p>All assignment due dates are listed in the user's specified timezone, defaulting to UTC (the international time standard) if one has not been chosen. EdX cannot offer any extensions.</p>
```
Comment (yro) [inline problem/9cee77a606ea4c1aa5440e0ea5d0f618.xml]: We will attempt to determine the user's system timezone, and failing that, will revert to UTC.

Comment (efischer19) [inline problem/9cee77a606ea4c1aa5440e0ea5d0f618.xml]: Wait, so how does the user's preference-selected timezone figure into that?

Comment (catong) [inline problem/9cee77a606ea4c1aa5440e0ea5d0f618.xml]: I'm confused too. Does user's "system timezone" = their account specified timezone or their browser timezone?

**Comment by efischer19** [comment]
New screenshot, includes @catong's updates but the resolution of the system time vs account specified time vs UTC discussion is not here.

screen shot 2016-12-14 at 1 19 48 pm

File: problem/9cee77a606ea4c1aa5440e0ea5d0f618.xml [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
   <p>Questions which are part of assignments or exams may have due dates - the last possible time you can submit an assignment for grading. Once this time has passed, you will not be able to get credit for any incomplete problems in the assignment.</p>
-  <p>If an assignment has a due date, you can see the due date in the accordion on the side. (This demo course does not have any due date.) If no due date is displayed, the assignment can be turned in at any time.</p>
-  <p>All assignment due dates are listed in UTC (the international time standard). Please <a href="http://www.timeanddate.com/worldclock/">look up your time zone</a> in order to understand what time your assignments are due according to your clock. EdX cannot offer any extensions.</p>
+  <p>If an assignment has a due date, you can see the due date in the sidebar. (This demo course does not have any assignments with due dates.) If no due date is displayed, the assignment can be turned in at any time.</p>
+  <p>All assignment due dates are listed in the user's preferred timezone, defaulting to the user's browser defined timezone if one has not been chosen. EdX cannot offer any extensions.</p>
```
Comment (efischer19) [inline problem/9cee77a606ea4c1aa5440e0ea5d0f618.xml]: @catong I feel like there's a better way to phrase this, but I'm not sure what it is

Comment (catong) [inline problem/9cee77a606ea4c1aa5440e0ea5d0f618.xml]: How about this: 
"All assignment due dates display in the time zone that you select in your account settings. If you do not specify a local time zone, assignment due dates display in your browser's time zone."
Also, I'd like to strike "EdX cannot offer any extensions." It's out of place and confusing here.

**Comment by catong** [comment]
Thanks! 👍

**Comment by efischer19** [comment]
Latest screenshot:

screen shot 2016-12-14 at 2 04 15 pm

**Pull request merged**